digitalpublicgoods/provenance-services-with-smart-data-agreements-using-data-exchange-protocols.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "Provenance Services with Smart Data Agreements using Data Exchange Protocols",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Unknown",
+    "copyrightURL": ""
```

Comment:
MIssing import copyright information, please check with product owner. Thanks.

Reply:
Pls view the info here: https://github.com/decentralised-dataexchange/data-exchange-agreements/blob/main/LICENSE